d2bs/kolbot/libs/common/Town.js

```diff
@@ -69,6 +69,12 @@ var Town = {
 	doChores: function () {
 		if (!me.inTown) {
 			this.goToTown();
+		} else if (me.area === 40 && me.x === 5153 && me.y === 5203) {
```

Comment:
instead of checking exact location, perhaps check distance from warriv < threshold -> use warriv